services/web/env.conf

```diff
@@ -6,4 +6,10 @@ BIND_PORT=2200
 APP_NAME=Bedrock
 API_URL=http://localhost:2300/
 
-SENTRY_DNS=
\ No newline at end of file
+SENTRY_DNS=
```

Comment:
I wonder if this should rather be called SENTRY_DSN

Reply:
Yes! I actually fixed that elsewhere but it never made it back... just updated.